src/test/java/org/opensearch/security/test/helper/rest/RestHelper.java

```diff
@@ -136,48 +137,48 @@ public HttpResponse[] executeMultipleAsyncPutRequest(final int numOfRequests, fi
 				.toArray(s -> new HttpResponse[s]);
 	}
 
-	public HttpResponse executeGetRequest(final String request, Header... header) throws Exception {
+	public HttpResponse executeGetRequest(final String request, Header... header) {
 	    return executeRequest(new HttpGet(getHttpServerUri() + "/" + request), header);
 	}
 	
-	public HttpResponse executeHeadRequest(final String request, Header... header) throws Exception {
+	public HttpResponse executeHeadRequest(final String request, Header... header) {
         return executeRequest(new HttpHead(getHttpServerUri() + "/" + request), header);
     }
 	
-	public HttpResponse executeOptionsRequest(final String request) throws Exception {
+	public HttpResponse executeOptionsRequest(final String request) {
         return executeRequest(new HttpOptions(getHttpServerUri() + "/" + request));
     }
 
-	public HttpResponse executePutRequest(final String request, String body, Header... header) throws Exception {
+	public HttpResponse executePutRequest(final String request, String body, Header... header) {
 		HttpPut uriRequest = new HttpPut(getHttpServerUri() + "/" + request);
 		if (body != null && !body.isEmpty()) {
-			uriRequest.setEntity(new StringEntity(body));
+			uriRequest.setEntity(createStringEntity(body));
```

Comment:
Why does `new StringEntity(body)` need to be moved to a separate method and throw `RuntimeException` from there?

Reply:
When creating runnable they cannot throw an exception, this allows calls to these methods to be more concise.

`() -> executePutRequest(...)` vs
`() -> { try { return executePutRequest(...); } catch (Exception e) { throw new RuntimeException(e);}`

While in a production workflow capturing these individual exceptions has utility, they are almost never expected to be handled in this test code so runtime exception is appropriate.